cmd/gitserver/server/server.go

```diff
@@ -1768,16 +1772,14 @@ func (s *Server) p4exec(w http.ResponseWriter, r *http.Request, req *protocol.P4
 
 		var tr *trace.Trace
 		tr, ctx = trace.New(ctx, "p4exec."+cmd, req.P4Port)
-		tr.LogFields(
-			otlog.Object("args", args),
-		)
+		tr.AddEvent("LogFields", attribute.String("args", args))
```

Comment:
These look like attributes as well:

```suggestion
		tr.SetAttributes(attribute.String("args", args))
```